client/my-sites/exporter/exporter.jsx

```diff
@@ -52,8 +53,16 @@ export default React.createClass( {
 		} = this.props;
 		const siteId = this.props.site.ID;
 
-		const exportAll = () => startExport( siteId );
-		const exportSelectedItems = () => startExport( siteId, { exportAll: false } );
+		const exportAll = () => {
+			analytics.tracks.recordEvent( 'calypso_export_start_click', { scope: 'all' } );
```

Comment:
I had a quick moment of wondering how one starts a click vs. finishing it. I think it was just me being morning stupid, but maybe `calypso_export_start_button_click` would be a hair more explicit and remove any possibility for confusion?

I also wondered if we shouldn't move the call to `recordEvent()` into `startExport()` since it would eliminate a repetition, and we could use the `exportAll` property to determine what scope to send. What you've done here may be better; I'm not sure; if so, discard this comment.

Tweak these things or not at your discretion, and 🚢  when ready.